paper-slider.html

```diff
@@ -474,7 +474,7 @@
       ready: function() {
         // issue polymer/polymer#1305
         this.async(function() {
-          this._updateKnob(this.value);
+          this._updateKnob(this.value, this.min, this.max);
         }, 1);
```

Comment:
I didn't really read the above mentioned issue (polymer/polymer#1305) - not sure if it's still relevant.